web/src/components/App/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { Route, Switch } from 'react-router-dom'
+import { ThemeProvider } from 'emotion-theming'
+import { theme } from 'theme'
```

Comment:
Using just the module name works in running the application. However, the tests are not able to follow the paths. This applies not just to `theme` but all the custom module imports. I talk about it more in issue #196.